packages/angular/projects/ui-angular/src/lib/primitives-poc/flex/flex-component/flex.component.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  Component,
+  ElementRef,
+  HostBinding,
+  Input,
+  Renderer2,
+} from '@angular/core';
```

Comment:
Can we remove unused imports? https://github.com/aws-amplify/amplify-ui/pull/2352#issuecomment-1197486251 tells you these!

Reply:
Done.